src/reducers/index.js

```diff
@@ -372,14 +372,13 @@ const onUpdateLoadedData = (state, action) => ({
 });
 
 const onCleanData = (state) => {
-  if (state.wallet) {
-    state.wallet.stop();
```

Comment:
Stopping the wallet emits a change of state, which emits an event to update the wallet state on redux.
Since stop is sync, we receive an error that we cannot emit an action from a reducer.
Also the wallet does not need to be stopped here.